app/plugins.js

```diff
@@ -70,6 +71,15 @@ function decorateObject(base, key) {
   return decorated;
 }
 
+exports.extendKeymaps = function () {
+  modules.forEach(plugin => {
+    if (plugin.extendKeymaps) {
+      const keys = _keys.extend(plugin.extendKeymaps());
+      config.extendKeymaps(keys);
```

Comment:
If we have not any other solution than `commandRegistry` global singleton, I think it is better to not expose it to plugins.
We should register callback here instead and not let plugin knows something about windows.commandRegistry.